server/controller/user.js

```diff
@@ -0,0 +1,94 @@
+const userService = require('../service/user');
+
+exports.create = (req, res) => {
+    const { body } = req;
+    userService.createUser(body, (error, response) => {
+        if (response) {
+            res.status(201).send(response);
+        } else if (error) {
+            res.status(400).send(error);
+        }
+    });
+};
+
+exports.find = (req, res) => {
+    const params = req.params || {};
+    const query = {
+        id: params.id
+    };
+    if (!query) {
+        res.status(400).send('Bad Request');
+        return;
+    }
+    userService.findUser(query, (error, response) => {
+        if (error) {
+            res.status(404).send(error);
+            return;
+        }
+        if (response) {
+            res.status(200).send(response);
+            return;
+        }
+        if (!response) {
+            res.status(204).send('No Data Found');
+        }
+    });
+};
+
+exports.findAll = (req, res) => {
+    const params = req.params || {};
+    console.log(params); // { sort: '-1', limit: '3' }
```

Comment:
When I do as GET /api/users/:sort/:limit
I actually get the params like
{ sort: '-1', limit: '3' }